emsdk.py

```diff
@@ -2652,8 +2653,7 @@ def expand_sdk_name(name):
   elif name == 'tot-upstream':
     return str(find_tot_sdk('upstream'))
   elif name in ('tot-fastcomp', 'sdk-nightly-latest'):
-    print('error: The fastcomp compiler is not available tot/nightly builds. Please use the upstream llvm backend or use an older version than 2.0.0.')
-    sys.exit(1)
+    exit_with_fastcomp_error('The fastcomp backend is not getting tot/nightly builds.')
```

Comment:
Just move these two up to the top and combine with the others?

Reply:
I don't want to combine them as the error text is a little different. I can move it up though, but then it would separate the `tot-*` ones from each other. What do you prefer?

Reply:
But why not just make the error slightly more generic:  "fastcomp is no longer available in recent releases."

If you add `The available sdk with fastcomp support is `fastcomp-1.40.1` I think that would be enough to make just one single error message that would work in all tree cases.. but i'm ok if you doing do that part.

Reply:
Sounds fine, I guess the error message doesn't need that much detail. I agree it's simpler code that way, done.